CONTRIBUTORS.md

```diff
@@ -152,4 +152,5 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 * [Rishabh Shah](https://github.com/rms13)
 * [Rudraksha Shah](https://github.com/Rudraksha20)
 * [Cody Guldner](https://github.com/burn123)
+* [Florent Cayré](https://github.com/fcayre)
```

Comment:
Can you please add yourself under the corporate CLA section?

Reply:
@pjcozzi done.